### PR TITLE
chore(package) update dependencies

### PR DESCRIPTION
**Pull request by 9renpoto**
Update Dependencies

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js:
-  - '6'
```
Comment (nickmccurdy) [inline .travis.yml]: We should still test 6, it's an active LTS.

File: package.json [hunk 2]
```diff
@@ -42,17 +42,17 @@
...
+    "ava": "^0.22.0",
+    "eslint": "^4.6.1",
+    "eslint-ava-rule-tester": "^2.0.2",
     "inject-in-tag": "^1.1.1",
-    "nyc": "^6.4.4",
-    "pify": "^2.3.0",
-    "xo": "^0.17.0"
+    "nyc": "^11.2.1",
+    "pify": "^3.0.0",
+    "xo": "^0.18.2"
```
Comment (nickmccurdy) [inline package.json]: It looks like a couple rules are now failing the linter build, let's fix them.

**Comment by 9renpoto** [comment]
@nickmccurdy Thank you review for my PR. Fixed it.

**Comment by 9renpoto** [comment]
This PR is too old. close it.

**Pull request closed without merging**